src/webui/www/private/scripts/client.js

```diff
@@ -621,10 +625,11 @@ window.addEvent('load', function() {
                     }
                     if (response['trackers']) {
                         for (const tracker in response['trackers']) {
+                            const host = getTrackerHost(tracker);
```

Comment:
Please move it below `const torrents ...`

Reply:
Done.

Reply:
?? I didn't see the update on github.

In case there was a misunderstanding, I meant:
```js
const torrents = response['trackers'][tracker];
const host = getTrackerHost(tracker);
const hash = genHash(host);
```

Reply:
Sorry, forgot to push my changes.